### ISSUE TITLE
Add Sidebar to Floorplan Template

### ISSUE THREAD

**Issue by kzadrozny** [uchicago-library/library_website#12]
Per this mock, approved by the Floorplan subgroup: https://kzadrozny.github.io/LibPattern/floorplan.html

Adds:

- Link to filtered view of "Spaces" on the floor
- Call number range on floor
- Collections on the floor
- Related floorplan (for reading room / bookstacks)

**Comment by kzadrozny** [comment]
Dependent on Issue #53

**Comment by kzadrozny** [comment]
The revised floorplans have merged the Reading Room and Bookstacks, allowing us to have one online floorplan for each floor. The [original mock](https://uchicago-library.github.io/LibPattern/floorplan.html) had a space for "Related floorplan (for reading room / bookstacks)," which is no longer needed. The group is considering using the space to list the staff who have offices on the respective floor.

I've updated the original issue to reflect these changes and to link to the updated floorplan template mock.
